quilt/tools/command.py

```diff
@@ -254,9 +255,18 @@ def build(package, path=None, dry_run=False, env='default'):
     """
     Compile a Quilt data package, either from a build file or an existing package node.
     """
-    # we may have a path, PackageNode, or None
+    # we may have a path, git URL, PackageNode, or None
     if isinstance(path, string_types):
-        build_from_path(package, path, dry_run=dry_run, env=env)
+        # is this a git url?
+        is_git_url = re.match('http[s]?://(?:[\w./~_-])+\.git$', path)
```

Comment:
Need an `r` in front of the regex. It happens to work - but in general, backslashes will break things.

Also, I'd move it into a constant and compile it, similar to e.g. `VALID_NAME_RE` in `quilt/tools/store.py`